app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    param = params.require(:comment).permit(:text)
+    @comment = Comment.new(author: @user, post: @post, text: param[:text])
```

Comment:
- [ ] A comment is created on behalf of the currently logged in user(current_user method in your application controller) of the app. In the requirements of this project, it is also stated that the comments controller should have `new` and `create` actions. Maybe, this piece of code below can assist you.

```ruby
 def new
    @comment = Comment.new
  end

 def create
    @comment = @current_user.comments.new(comment_params)
      ..
      ..
```

Where comment_params is a private method for security aspects as mentioned above

Reply:
Thanks for the review @FrederickMih, I think the new method for comment wasn't in the requirement and